### PR TITLE
athb & athb_array

### PR DESCRIPTION
**Pull request by Lu-Whale**
implemented athb and athb_array

### CODE REVIEW

File: src/models/athb.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+ * @param { number } t_running_mean - running mean temperature, in [°C]
+ *
+ * The running mean temperature can be calculated using the function
+ * jsthermalcomfort.utilities.running_mean_outdoor_temperature.
+ *
+ * @returns { number } athb_pmv - Predicted Mean Vote calculated with the Adaptive Thermal Heat Balance framework
+ *
+ * @example
+ * // Import the athb function from the jsthermalcomfort module
+ * import { athb } from "./models/athb.js";
```
Comment (szaldivar) [inline src/models/athb.js:40]: does matter right now but the examples should import from the public library

File: src/models/athb.js [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+      0.000485 * t_running_mean +
+      0.08176 * met_adapted -
+      0.00527 * t_running_mean * met_adapted,
+  );
+
+  const pmv_res = pmv_calculation(tdb, tr, vr, rh, met_adapted, clo_adapted, 0);
+  const ts = 0.303 * Math.exp(-0.036 * met_adapted * 58.15) + 0.028;
+  const l_adapted = pmv_res / ts;
+
+  return parseFloat(
```
Comment (szaldivar) [inline src/models/athb.js:60]: use round from utilities

File: src/models/athb.js [hunk 3]
```diff
@@ -0,0 +1,163 @@
+import { pmv_calculation } from "./pmv_ppd.js";
+
+/**
+ * Return the PMV value calculated with the Adaptive Thermal Heat Balance
+ * Framework [27]_. The adaptive thermal heat balance (ATHB) framework
```
Comment (szaldivar) [inline src/models/athb.js:5]: add a reference in references.md and link to it here

**Review by szaldivar — CHANGES_REQUESTED** [review]

File: src/models/athb.js (hunk 1 above)
Comment (Lu-Whale) [reply src/models/athb.js]: deleted

File: src/models/athb.js (hunk 2 above)
Comment (Lu-Whale) [reply src/models/athb.js]: changed

File: src/models/athb.js (hunk 3 above)
Comment (Lu-Whale) [reply src/models/athb.js]: added

**Comment by szaldivar** [comment]
this seems okay now, I will approve once all the checks pass

**Review by szaldivar — APPROVED** [review]

File: tests/index.test.js [hunk 4]
```diff
@@ -108,6 +108,13 @@ describe("NPM Package", () => {
 
   it("should have models.athb_array", () => {
     expect(jsthermalcomfort.models).toHaveProperty("athb_array");
+
```
Comment (szaldivar) [inline tests/index.test.js:111]: sneaky missing brackets

**Pull request merged**